libs/db-browser/src/main/resources/schema/sql/oboracle/oboracle_4_1_x.yaml

```diff
@@ -0,0 +1,277 @@
+sqls:
+  list-basic-table-columns: |-
+    SELECT
+      OWNER,
+      TABLE_NAME,
+      COLUMN_NAME,
+      DATA_TYPE,
+      COMMENTS
+    FROM
+      SYS.ALL_TAB_COLS NATURAL JOIN SYS.ALL_COL_COMMENTS
+    WHERE
+      OWNER = ? AND TABLE_NAME = ? AND USER_GENERATED='YES'
```

Comment:
so, I get two questions:
1. is column 'USER_GENERATED' a new column after OB 4.1.0 and not appear before OB 4.1.0?
2. why are 'list-basic-view-columns' and 'list-basic-schema-view-columns' not filtered by this column?